barretenberg/cpp/src/barretenberg/stdlib/primitives/biggroup/biggroup_impl.hpp

```diff
@@ -127,6 +127,24 @@ element<C, Fq, Fr, G> element<C, Fq, Fr, G>::operator+(const element& other) con
     return result;
 }
 
+/**
+ * @brief Enforce (0,0) x y in the case of point at infinity
+ *
+ * @details We need to have a standard witness in Noir and the point at infinity can have non-zero random coefficients.
```

Comment:
I don't really understand this sentence.. you mean to prevent the point at infinity from having random coefficients?